syntax = "proto3";
package ChatVllm;

service LlmEngine {
        rpc processChatReq(ChatReq) returns (ChatResp){}
}

message ChatReq {
        string prompt = 1;
        string request_id = 3;
}

message ChatResp {
        string answer = 1;
}